test/unit/init/CliRunner.test.ts

```diff
@@ -179,7 +179,54 @@ describe('CliRunner', (): void => {
 
     expect(write).toHaveBeenCalledTimes(0);
 
-    expect(exit).toHaveBeenCalledTimes(1);
     expect(exit).toHaveBeenCalledWith(1);
   });
+
+  it('exits when unknown options are passed to the main executable.', async(): Promise<void> => {
+    new CliRunner().run({
+      argv: [
+        'node', 'script', '--foo',
+      ],
+    });
+
+    // Wait until initializer has been called, because we can't await CliRunner.run.
+    await new Promise((resolve): void => {
+      setImmediate(resolve);
+    });
+
+    expect(exit).toHaveBeenCalledTimes(1);
+    expect(exit).not.toHaveBeenCalledWith(0);
```

Comment:
```suggestion
    expect(exit).toHaveBeenCalledWith(1);
```
(because it could also have been called with `undefined`, which we don't want)